src/scenarios/miner_std.py

```diff
@@ -26,7 +27,7 @@ def run_test(self):
             miner = ensure_miner(self.nodes[current_miner])
```

Comment:
would be nice to have a controller container who's job it was to run scenarios like this, so that the docker host  could disconnect and leave things running